Gemfile.jruby-1.9.lock

```diff
@@ -503,7 +503,7 @@ GEM
     murmurhash3 (0.1.6-java)
     mustache (0.99.8)
     naught (1.1.0)
-    nokogiri (1.6.8-java)
+    nokogiri (1.6.8.1-java)
```

Comment:
not sure. I ran a bundle update, so just a new version. I supposed its harmless since it's a point release..